twine/wheel.py

```diff
@@ -86,6 +86,6 @@ def read_file(name: str) -> bytes:
     def parse(self, data: bytes) -> None:
         super().parse(data)
 
-        fp = io.StringIO(distribution.must_decode(data))
+        fp = io.StringIO(data.decode("utf-8", errors="replace"))
```

Comment:
Given the original function here seems to have been used mostly for 2.X sake, I went with just calling `decode`, and since the original function fell back to `latin-1`, I went with `errors=replace`. But I'm not sure the latter case can actually happen in real life (i.e. whether it'd be against the wheel spec anyhow)?